reports-rest/src/main/java/org/jboss/da/rest/listings/Artifacts.java

```diff
@@ -168,16 +172,18 @@ public Response addWhiteArtifact(
                     response.setSuccess(false);
                     return Response
                             .status(Response.Status.CONFLICT)
-                            .entity(new ErrorMessage(
+                            .entity(new ErrorMessage(ErrorMessage.eType.BLACKLIST,
                                     "Can't add artifact to whitelist, artifact is blacklisted"))
                             .build();
                 case NOT_MODIFIED:
                     response.setSuccess(false);
                     return Response.ok(response).build();
                 default:
                     response.setSuccess(false);
-                    return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
-                            .entity(new ErrorMessage("Unexpected server error occurred.")).build();
+                    return Response
+                            .status(Response.Status.INTERNAL_SERVER_ERROR)
+                            .entity(new ErrorMessage(ErrorMessage.eType.UNEXPECTED_SERVER_ERR,
+                                    "Unexpected server error occurred.")).build();
```

Comment:
There are two messages for internal server error "There was some internal error"  and "Unexpected server error occurred" I think second sounds better, can you use it on other places too? @jbrazdil?